boards/arm/cyclonev_socdk/Kconfig.defconfig

```diff
@@ -28,4 +28,14 @@ config SPI_DW_FIFO_DEPTH
 	default 256
 endif # SPI_DW
 
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y
+config TEST_RANDOM_GENERATOR
+	default y
+config ETH_CYCLONEV
+	default y
```

Comment:
This won't be needed after you make use of `DT_HAS_SNPS_ETHERNET_CYCLONEV_ENABLED`